docs/UserGuide.md

```diff
@@ -108,6 +108,18 @@ Displays a list of all the positions stored in the application.
 
     list_p
 
+<u>**Edit a position: `edit_p`**</u>
+
+Edits a specific position's details. Only one edit field is needed. Users cannot edit both fields.
+
+<u>Format:</u>
+
+    edit_p <INDEX> [title=<TITLE>] [status=<STATUS>]
+
+<u>Example:</u>
+`edit_P 3 status=closed`
+* Edits the status of the 3rd position in the list to closed.
+
```

Comment:
Maybe can add on to say what happens when status is set to closed? Like it removes the position from every candidate.